### ISSUE TITLE
Removing Medals

### ISSUE THREAD

**Issue by raveious** [cremalab/slack-scorebot#13]
Our team has a vast amount of custom emoji that we added to our team. It would be great to remove the old ones or remove ones that were added accidentally.

something like...
`@scorebot delete :tada:`

**Comment by ChrisWphoto** [comment]
I agree! This is a feature that needs to be implemented. This combined with better RegExp matching could really help provide a cleaner `show medals` experience. 

I will start work on this soon. 

Anything else you like/dislike about the experience?

**Comment by raveious** [comment]
Somehow be able to limit who can edit the medals to any member who is a team admin or owner. I'm pretty sure this can be done via the [users.info](https://api.slack.com/methods/users.info) method.

**Comment by ChrisWphoto** [comment]
Great feedback. I've added it to our slack channel, I'll let you know how this develops.

**Comment by raveious** [comment]
Alternatively, instead of adding a command to remove medals. How about setting them to 0 points will delete them?
